resources/views/projects/create.blade.php

```diff
@@ -0,0 +1,252 @@
+@extends('layouts.app')
+
+@section('title', 'Add Project')
+
+@section('content')
+
+@section('header', 'Register a new project')
+<style>
+	#or {
+		text-align: center;
+		color: #C7C7C7;
+	}
+</style>
+<div class="padded content">
+	<form class="ui form" method="POST" action="/projects">
+		@csrf
+		<!-- Project Image -->
+		<div class="projectImage field">
+			<label for="projectImage">Project Image</label>
+			<div class="projectImageUploaderContainer">
+				<div class="imgUploader" class="imagePreviewContainer">
+					<img src="https://lorempixel.com/400/400" alt="Project Image" class="ui medium image" id="projectImagePreview"/>
+				</div>
+				<a href="#" class="imgUploader">
```

Comment:
No es necesario poner un link y luego un bot&oacute;n. Mejor elige uno de los dos and stick to that (para este escenario yo eligir&iacute;a el bot&oacute;n)

Reply:
tienes razón, listo!